enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -48,7 +48,9 @@
   [user-id]
   (let [roots (t2/select Collection {:where [:and [:= :location "/"]
                                                   [:or [:= :personal_owner_id nil]
-                                                       [:= :personal_owner_id user-id]]]})]
+                                                       [:= :personal_owner_id user-id]]
+                                                  [:or [:= :namespace nil]
+                                                       [:!= :namespace "analytics"]]]})]
```

Comment:
I don't think `[:= :namespace nil]` is necessary, right?

Reply:
and could you please update the doc of this function too?

Reply:
It is, since `!=` will skip all NULL entries. So it's either that, or using `is distinct from`.

Reply:
> and could you please update the doc of this function too?

I can, no probs, but it feels still relevant to me as it is.

Reply:
huh, so TIL then!